### DISCUSSION TITLE
Can Spatial Temporal Action Detection SlowFast model predict bbox？

### DISCUSSION THREAD

**Discussion by ZJU-lishuang** [open-mmlab/mmaction2#711] [Q&A] [answered]
Can Spatial Temporal Action Detection SlowFast model predict bbox？
If it can predict bbox,how to do it?

**Comment by irvingzhang0512** [comment, marked as answer]
No. bboxes are the inputs of spatial temporal action detection model.

**Reply by ZJU-lishuang** [reply]
Is there a better model than slowfast in Spatial Temporal Action Detection?
bboxes as the inputs of spatial temporal action detection is a sad thing.

**Reply by irvingzhang0512** [reply]
> Is there a better model than slowfast in Spatial Temporal Action Detection?
> bboxes as the inputs of spatial temporal action detection is a sad thing.

In order to get better bboxes, we'd better use another model to detect humans.
For now, slowfast is the best in mmaction2

**Reply by ZJU-lishuang** [reply]
Is there a better model in Spatial Temporal Action Detection？
not only in mmaction2.
I didn't find one.

**Reply by ZJU-lishuang** [reply]
MOC-detector is not good in ava dataset.

**Reply by irvingzhang0512** [reply]
try alphaction, good-looking, hard to train and install.

**Reply by ZJU-lishuang** [reply]
Now I try to convert custom dataset to ava format.I got a csv like ava.
And I find the csv file already has the box coordinates.
What is the function of proposals in ava_dense_proposals_train.FAIR.recall_93.9.pkl during training slwofast?
